test/test-contrast.js

```
@@ -0,0 +1,166 @@
+const fs = require('fs');
+const path = require('path');
+
+const THEME_DIR = path.join(__dirname, '..', 'themes');
+const MIN_CONTRAST_RATIO = 4.5; // WCAG AA standard for normal text
+const MIN_CONTRAST_RATIO_LARGE = 3; // WCAG AA standard for large text
+
+// Convert hex to RGB
+function hexToRgb(hex) {
+  const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex);
+  return result ? {
+    r: parseInt(result[1], 16),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)